src/__tests__/Select-spec.jsx

```diff
@@ -1,39 +1,139 @@
 jest.unmock('../Select');
+jest.unmock('../SelectOption.js');
 
 // Don't need to test these and they currently throw errors
 jest.setMock('ship-components-outsideclick', 'div');
 jest.setMock('ship-components-highlight-click', 'div');
 
-
 import React from 'react';
-import TestUtils from 'react-addons-test-utils';
+import TestUtils from 'react-dom/test-utils';
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
 
 describe('Select', () => {
-  let itemsProps = ['One', 'Two', 'Three'];
+  let options = ['One', 'Two', 'Three'];
   let Select;
+  let SelectOption;
   beforeEach(() => {
     Select = require('../Select').default;
+    SelectOption = require('../SelectOption').default;
   });
 
-  // Render without error
-  it('should render without error', () => {
+  it('renders without error', () => {
     let element = React.createElement(
-       Select, // component class
+       Select,
        {
-        options: itemsProps
-       } // props go here
+        options: options
+       }
     );
 
     expect(() => TestUtils.renderIntoDocument(element))
        .not.toThrow();
   });
 
-  it('should support custom css classes', () => {
+  it('defines refs to input and list nodes', () => {
```

Comment:
What does testing if the refs get set do? What's the functionality here you are testing?

Reply:
Good point. Other tests that assert functionality should implicitly handle this case.